MozillaFoundation/foundation.mozilla.org
# truncating breadcrumbs

# Description


Related PRs/issues: #10440
Link to sample test page: https://foundation-s-10440-trun-h9yzip.herokuapp.com/en/research/library/power-book-view-usually-throughout/


This PR updates the template `breadcrumbs.html` by adding new classes to the breadcrumb element,  which will truncate the word with an ellipsis in the situation where the word will not fit.

# Screenshots
## Before:
<img width="400" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/7f424dcd-3e44-4f48-9dc7-9df238da9eef">


## After:

**Same breadcrumbs as before:**

<img width="400" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/960be58f-ba9a-4f41-a6e7-5aa73ff7e18f">





**Extra breadcrumbs for testing:**

<img width="400" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/9b03b869-8732-492e-90e6-e254285c0513">


# Steps to test:

1. Visit the test page linked above
2. Resize the page and make sure that the breadcrumbs are being truncated as expected on smaller screen sizes
3. If everything is working as expected, testing is complete!
